reference/errors/error_codes.mdx

```diff
@@ -142,6 +142,10 @@ The provided [document identifier](/learn/core_concepts/primary_key#document-id)
 
 The [`fields`](/reference/api/documents#query-parameters) parameter is invalid. It should be a string.
 
+## `invalid_document_filter`
+
+The [`filter`](/reference/api/documents#query-parameters) parameter is invalid. It should be a string, array of strings, or array of array of strings.
```

Comment:
This error code can also be returned if the filter can't be parsed.
Also in the case the get route was used it can only be a strings :grimacing: 
tbh I don't really know how to document it, maybe we should just drop a link to this page; https://www.meilisearch.com/docs/learn/advanced/filtering#configuring-filters

Reply:
I'll add more details! 

I didn't get this:
>Also in the case the get route was used it can only be a strings  

Do you mean we can only use strings with `filter` for GET `/indexes/{index_uid}/document`?
The following query worked for me:
```
 curl \
    -X GET 'http://localhost:7700/indexes/books/documents?limit=2&filter=id=2'
```
`id` is an integer for this dataset

Reply:
Hey, yes, almost :grin: 
I'm saying that the whole filter can only be a string.
In Meilisearch the filters can be either a string, an array of string,s or an array of array of strings.
But in the case of the get request, it can only be a string.
Our array syntax (with the comma) that we use for the other route like `/tasks?ids=1,2,3` doesn't works for these filter, i.e.: `../documents?filter=id=2,id=3` is invalid. You must write `../documents?filter=id=2 OR id=3` (and you must encode the space correctly).

Reply:
>Our array syntax (with the comma) that we use for the other route like /tasks?ids=1,2,3 doesn't works for these filter, i.e.: ../documents?filter=id=2,id=3 is invalid. You must write ../documents?filter=id=2 OR id=3 (and you must encode the space correctly).

Thanks for clarifying!

Reply:
- Updated the error code to add more details: lines 145-154 in `reference/errors/error_codes.mdx` 
- Updated the GET endpoint to add more details on `filter`:  line 94 in `reference/api/documents.mdx`